code/style.css

```diff
@@ -1,7 +1,215 @@
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background: lightpink;
+  color: black;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.header {
+  background-image: url("header_small.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 10px;
+  padding: 18px;
+  border: 4px solid lightgray;
+  border-radius: 20%;
+  display: inline-flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  height: 500px;
+  width: 100%;
+}
+
+.primary-header {
+  text-shadow: 2px 2px lightpink;
+}
+
+.navigation-bar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  margin-top: 200px;
+}
+
+.navigation-bar a:link,
+a:visited {
+  color: black;
+  padding: 8px 12px;
+  text-decoration: none;
+  text-align: center;
+}
+
+.navigation-bar a:hover {
+  color: deeppink;
+}
+
+.placeholder {
+  margin: 10px;
+  padding: 18px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 1300px;
+  width: 100%;
+}
+
+.form {
+  background: lightgray;
+  padding: 48px;
+  margin: 10px;
+  border: 4px solid gray;
+  border-radius: 20%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  text-align: left;
+  height: 700px;
+  width: auto;
+  object-fit: cover;
+  box-shadow: 20px 20px gray;
+}
+
+input {
+  margin: 8px;
+  padding: 5px;
+}
+
+.form-button {
+  width: 40%;
+  height: 30px;
+  margin: 8px;
+  padding: 5px;
+}
+
+.accordion {
+  background: lightgray;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin: 10px;
+  margin-right: auto;
+  padding: 18px;
+  height: 550px;
+  width: auto;
+  border: 4px solid gray;
+  border-radius: 20%;
+  object-fit: cover;
+  box-shadow: 20px 20px gray;
+}
+
+.accordion-header {
+  color: black;
+  cursor: pointer;
+  margin: 10px;
+  padding: 18px;
+}
+
+.accordion-content {
+  display: none;
+  color: black;
+  cursor: pointer;
+  margin: 10px;
+  border-top: 2px solid lightgray;
+  padding: 18px;
+  text-align: left;
+  outline: none;
+  transition: 0.4s;
+}
+
+.accordion-selected {
+  display: block;
+}
+
+.footer {
+  display: flex;
+  background: lightgray;
+  margin: 10px;
+  padding: 18px;
+  text-align: left;
+  border: 4px solid gray;
+  border-radius: 20px;
+  height: 200px;
+  width: 100%;
+}
+
+.footer-text {
+  font-weight: bold;
+}
+
+@media (min-width: 667px) and (max-width: 1024px) {
+  .header {
+    background-image: url("header_medium.jpg");
+    border-radius: 0px;
+  }
+
+  .navigation-bar {
+    margin-top: 130px;
+  }
+
+  .placeholder {
```

Comment:
Consider a column layout or allocating some more space as it's looking a little squished in a small tablet:

<img width="573" alt="Skärmavbild 2021-02-05 kl  15 01 23" src="https://user-images.githubusercontent.com/51529201/107043239-067bea80-67c3-11eb-8029-29d4f13dcd99.png">

Reply:
You are totally right, changed this to column.